refs/schemas/priv/RFC-PRIV.md

```diff
@@ -0,0 +1,893 @@
+
+    -  **<a name="target-terms"></a>Target Terms**: {`ORGANIZATION`, `PARTNERS`, `SYSTEM`} or any of their subcategories defined according to [Term Dot Notation](#term-dot-notation). *See definitions in the [dictionary/targets](./dictionary/targets).*
+
+    -  **<a name="target-directions"></a>Target Direction Terms**: {`PARTNERS.DOWNWARD`, `PARTNERS.UPWARD`} or any of their subcategories defined according to [Term Dot Notation](#term-dot-notation). *See definitions in the [dictionary/targets](./dictionary/targets).*
+
+    - **<a name="statuses"></a>Status Terms**: {`GRANTED`, `DENIED`, `PARTIALLY-GRANTED`, `UNDER-REVIEW`} or any of their subcategories defined according to [Term Dot Notation](#term-dot-notation). *See definitions in the [dictionary/statuses](./dictionary/statuses).*
+
+    - **<a name="motives"></a>Motive Terms**: {`IDENTITY-UNCONFIRMED`, `LANGUAGE-UNSUPPORTED`, `VALID-REASONS`, `IMPOSSIBLE`, `NO-SUCH-DATA`, `REQUEST-UNSUPPORTED`, `USER-UNKNOWN`} or any of their subcategories defined according to [Term Dot Notation](#term-dot-notation). *See definitions in the [dictionary/motives](./dictionary/motives).*
+
+    - **<a name="boolean"></a>Boolean Terms**: {`YES`, `NO`} or any of their subcategories defined according to [Term Dot Notation](#term-dot-notation). *See definitions in the [dictionary/boolean](./dictionary/boolean).*
+
+    - **<a name="legal-bases"></a>Legal Base Terms**: {`CONTRACT`, `CONSENT`, `LEGITIMATE-INTEREST`, `NECESSARY`, `NECESSARY.LEGAL-OBLIGATION`, `NECESSARY.PUBLIC-INTEREST`, `NECESSARY.VITAL-INTEREST`, `OTHER-LEGAL-BASE`} or any of their subcategories defined according to [Term Dot Notation](#term-dot-notation). *See definitions in the [dictionary/legal-bases](./dictionary/legal-bases).*
+
+    - **<a name="retentions"></a>Retention Terms**: {`NO-LONGER-THAN`, `NO-LESS-THAN`} or any of their subcategories defined according to [Term Dot Notation](#term-dot-notation). *See definitions in the [dictionary/retentions](./dictionary/retentions).*
+
+    - **<a name="events"></a>Event Terms**: {`CAPTURE-DATE`,`RELATIONSHIP-END`, `SERVICE-END`} or any of their subcategories defined according to [Term Dot Notation](#term-dot-notation). *See definitions in the [dictionary/events](./dictionary/events).*
+
+### Privacy Request
+
+Data Subject is the author of a Privacy Request.
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `data-subject` |  1-* | [Data Subject Identities](#decentralized-identity-of-data-subjects) each containing one `dsid` and one `dsid-schema`|
+
+A Privacy Request is made by one and only one Data Subject.
+
+A System MAY have multiple ways to identify the Data Subject, especially when data about them came from some other System that uses different identifiers.
+
```

Comment:
```suggestion

> Let us imagine two Systems A and B interoperate, and exchange data about Bob. System A identifies bob with an ID (ID-a), System A sends Bobs data to System B. Systems B uses a different ID (ID-b).

> If Systems A and B want to interoperate, and be able to interpret Bobs future Privacy Requests, then:

> - System B MUST (in addition to ID-b, also save ID-a)
> - System that captures Bobs data MUST obtain consent for processing-category:MATCHING, target:PARTNERS
> - When Bob makes Privacy Request at System B but wants the Privacy Request transmitted to System A, System B must include both DSIDs (ID-a & ID-b) in the Privacy Request sent to System A
> - If Bob makes a REVOKE-CONSENT processing-category:MATCHING, then System B MUST delete the correspondence between ID-a and ID-b, and there is no more interoperability of future Privacy Requests.

> If System A and Systems B use the same ID for Bob, then all this is much simpler.

```

Reply:
> - If Bob makes a REVOKE-CONSENT processing-category:MATCHING, then System B MUST delete the correspondence between ID-a and ID-b, and there is no more interoperability of future Privacy Requests.

It's not really a MUST. The System MAY have legal base (Legitimate interest) do the matching even after revoked consent. I am accepting your change but will nuance the language around MUST a little bit.